src/components/compress/index.tsx

```diff
@@ -226,12 +228,23 @@ export default class Compress extends Component<Props, State> {
     });
   }
 
+  private updateDocumentTitle(filename : string = '') {
```

Comment:
No space before the `:`, so `filename: string = ''`. Sorry, I thought our linter would catch this.

Reply:
Also, when is filename empty?

Reply:
Ohh I see, it's on unmount. Gotcha.